### ISSUE TITLE
v1.7.0: Addon disabled for all domains by default?

### ISSUE THREAD

**Issue by stefan-muc** [wtimme/firefox-remove-safelinks#14]
Thanks for this neat addon!

Starting with version 1.7.0 the addon is broken as it is disabled for any destination. That change hit me by surprise as I now again get the 
![image](https://github.com/user-attachments/assets/63ed1a06-e4a4-48ba-ae08-9473b6027321)
messages and have to do a second click to view the link.

As a workaround I entered "http" in the list of explicitly specified domains. But as far as I understand the explanation there if I leave this box empty (which it is after update), addon is enabled for all destinations. But that's not the case with my installation (Firefox 132.0 (64-bit))

**Comment by LukeCz** [comment]
Thank you for submitting the bug. Yes, the empty box (default) should allow all sites. I did tested this several times just now and cannot reproduce this behavior, even tried changing the extension ID so it installs completely fresh.
I'm wondering if this is not actually connected with #15 and the background script stopping. I did check that updating options starts the script again so maybe this is what it was temporarily happening once you entered "http".

**Comment by wtimme** [comment]
Thanks for your report! I am going to take a look at this. For the time being, I have decided to revert the changes from 1.7.0, as to make sure that your privacy is preserved.

**Comment by stefan-muc** [comment]
I had a short look at the source before writing my report but didn't find anything obvious.
Yeah indeed it could be that I've been hit by #15. After writing my bug report, a second computer got updated and I "knew" that I had to set `http` as explicitly specified, so I did. But some time later, the extension also stopped there.
And as I'm writing this (and testing again) extension on first computer - not yet updated to 1.8.0 - also has stopped. This time just updating options didn't help, but I had to disable and re-enable the extension. So I guess you are right and I'll close this report and update to 1.8.0
